testing/internal/e2e/boundary/target.go

```diff
@@ -62,13 +62,25 @@ func CreateNewTargetCli(t testing.TB, ctx context.Context, projectId string, def
 }
 
 // AddHostSourceToTargetCli uses the cli to add a host source (host set or host) to a target
-func AddHostSourceToTargetCli(t testing.TB, ctx context.Context, targetId string, hostSourceId string) {
+func AddHostSourceToTargetCli(t testing.TB, ctx context.Context, targetId, hostSourceId string) {
 	output := e2e.RunCommand(ctx, "boundary",
 		e2e.WithArgs("targets", "add-host-sources", "-id", targetId, "-host-source", hostSourceId),
 	)
 	require.NoError(t, output.Err, string(output.Stderr))
 }
 
+// RemoveHostSourceFromTargetCli uses the cli to remove a host source (host set or host) to a target
+func RemoveHostSourceFromTargetCli(t testing.TB, ctx context.Context, targetId, hostSourceId string) (string, error) {
+	output := e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"targets", "remove-host-sources",
+			"-id", targetId,
+			"-host-source", hostSourceId,
+		),
+	)
+	return string(output.Stdout), output.Err
+}
```

Comment:
Should this take a `t testing.TB` if it's returning the error? I'd say it should either take a `t testing.TB` and then check for the error with `require`, or return the error and no accept a `t`. What do you think? It looks like we could just check the error in this helper function instead?

Reply:
Agreed. The pattern we've used so far is to do the error checking in the helper function. You can see `AddHostSourceToTargetCli` as an example.

Reply:
I agree, we should keep the pattern in the helper functions consistent.